src/envoy/http/path_matcher/filter_fuzz_test.cc

```diff
@@ -61,6 +62,12 @@ DEFINE_PROTO_FUZZER(
     // Run data against the filter.
     ASSERT_NO_THROW(doTest(filter, input));
 
+    // Ensure the query param filter state is valid.
+    absl::string_view query_params = utils::getStringFilterState(
+        *mock_decoder_callbacks.stream_info_.filter_state_,
+        utils::kQueryParams);
+    ASSERT_TRUE(Envoy::Http::validHeaderString(query_params));
```

Comment:
Are you always setting QueryParams?

Reply:
No, but `utils::getStringFilterState` will return empty string when it's not set. Empty string is valid.